src/js/modules/Contact/Address.js

```diff
@@ -2,20 +2,20 @@ import React from 'react';
 import MapMarkerSVG from 'js/svg/MapMarker';
 import PropTypes from 'prop-types';
 
-const Address = ({ name, location }) => (
-  <div className="coa-ContactDetails__item coa-ContactAddress">
+const Address = ({ location }) => (
+  <div className="coa-ContactItem coa-ContactAddress">
     <MapMarkerSVG size="20"/>
     <div>
-      <span>{name}</span>
+      <span>{location.name}</span>
```

Comment:
We have a name property in the location object and a name property in the contact object. Not sure why. I think this is just a legacy Joplin data thing. Which tells me we should take a minute to really review our data structures. Anywaaaaaaaay, I decided to just stick to the name property in the location object.